app/modules/parsing/graph_construction/code_graph_service.py

```
@@ -184,6 +212,61 @@ def create_and_store_graph(self, repo_dir, project_id, user_id):
+                        alias_name=collection_alias,
+                    )
+                    qdrant_index_time = time.time() - qdrant_index_start
+                    logger.info(
+                        f"[GRAPH GENERATION] Qdrant indexing completed: {count} points indexed, {n_bm25} BM25 tokens, {n_colbert} ColBERT tokens in {qdrant_index_time:.2f}s",
+                        project_id=project_id,
+                        indexed_count=count,
+                        bm25_tokens=n_bm25,
+                        colbert_tokens=n_colbert,
+                        qdrant_index_time_seconds=qdrant_index_time,
+                    )
+                except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)